### PR TITLE
feat: adicionando template de generators

### PR DESCRIPTION
**Pull request by ramonxm**
## Description

Organizando os generators com plop.

### CODE REVIEW

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: generators/templates/stories.tsx.hbs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Story, Meta } from '@storybook/react'
```
Comment (jonataspinto) [inline generators/templates/stories.tsx.hbs:1]: Faz sentido? Não temos storybook

**Review by jonataspinto — CHANGES_REQUESTED** [review]

File: generators/templates/stories.tsx.hbs (hunk 1 above)
Comment (ramonxm) [reply generators/templates/stories.tsx.hbs]: Acabei esquecendo, pode fazer sentido em algum momento mas é fácil recolocar novamente. Subi as changes.

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by jonataspinto — APPROVED** [review]

**Comment by jonataspinto** [comment]
@ramonxm merge bloqueado, pode verificar?

**Comment by ramonxm** [comment]
> @ramonxm merge bloqueado, pode verificar?

consigo mergear

**Pull request merged**